api/src/main/java/io/opentelemetry/api/trace/TraceState.java

```diff
@@ -170,15 +188,16 @@ public TraceState build() {
   /** Immutable key-value pair for {@code TraceState}. */
   @Immutable
   @AutoValue
-  public abstract static class Entry {
+  abstract static class Entry {
```

Comment:
You can file a follow-up to remove this and keep two vectors.

Reply:
Sure #1981